cpu/cpu_test.go

```diff
@@ -78,16 +78,20 @@ func createMockCPUInfo(dataSetNumber int) {
 			cpu1 3516068 1019269 190413 49493320 11620 0 278 0 0 0
 			intr 33594809 19 2 0 0 0 0 0 9 1 4 0 0 4 0 0 0 31 0 0`
 	} else if dataSetNumber == 2 {
```

Comment:
Which data set number is correct and which is incorrect? 
Adding new data sets might be error prone. 
Consider adding a boolean flag for clarity or otherwise indicating what's the intention of test. Perhaps hold the sample data in two separate slices.